### PR TITLE
Rocks 994 ca cert mutation scripts 22.04

### PR DESCRIPTION
**Pull request by zhijie-yang**
# Proposed changes

The script `update-ca-certificates` and its corresponding config file `/etc/ca-certificates.conf` is added to the SDF of `ca-certificates`.

## Related issues/PRs
ROCKS-994

### Forward porting

#266 

## Checklist

* [x] I have read the [contributing guidelines](
https://github.com/canonical/chisel-releases/blob/main/CONTRIBUTING.md)
* [x] I have already submitted the [CLA form](
https://ubuntu.com/legal/contributors/agreement)

### CODE REVIEW

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]
can u pls update this according to #266

**Review by cjdcordeiro — APPROVED** [review]

File: slices/ca-certificates.yaml [hunk 1]
```diff
@@ -4,18 +4,56 @@ essential:
...
       content.write("/etc/ssl/certs/ca-certificates.crt", "".join(certs))
 
+  bins:
+    essential:
+      - openssl_bins
+      # The `/usr/sbin/update-ca-certificates` script requires sed to run.
+      - sed_bins
+    contents:
+      # To run update-ca-certificates without breaking the default certificates
+      # at /etc/ssl/certs/ca-certificates.crt, the `_data-with-certs` slice should
```
Comment (cjdcordeiro) [inline slices/ca-certificates.yaml:53]: this slice doesn't exist

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]

File: slices/ca-certificates.yaml (hunk 1 above)
Comment (zhijie-yang) [reply slices/ca-certificates.yaml]: Fixed. Ref #285

**Review by cjdcordeiro — APPROVED** [review]

**Review by linostar — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**